test/automated/reflection/parameters/method_is_not_implemented.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../../automated_init'
+
+context "Reflect" do
+  context "Reflection" do
+    context "Parameters" do
+      context "Method Is Not Implemented" do
+        subject = Reflect::Controls::Subject.example
+
+        reflection = Reflect.(subject, :SomeConstant, strict: true)
+
+        random_method_name = Reflect::Controls::Namespace::Random.get
+
+        test "Is an error" do
+          assert_raises(Reflect::Error) do
+            reflection.parameters(random_method_name)
+          end
+        end
+      end
+    end
+  end
+end
```

Comment:
Consider: after target_method is implemented with test coverage for when the method isn't implemented, remove this file since it's test coverage is redundant. Expect to have just a single `parameters.rb` test file with no `parameters` directory afterwards.